extensions/amp-recaptcha-input/validator-amp-recaptcha-input.protoascii

```diff
@@ -13,3 +13,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the license.
 #
+tags: {  # amp-recaptcha-input
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-recaptcha-input"
+    version: "0.1"
+    version: "latest"
+    deprecated_allow_duplicates: true
```

Comment:
This allows the user to insert the extension script tag multiple times in the document. I don't think you want that, so I'd remove this line.